android/titanium/src/java/org/appcelerator/titanium/TiBaseActivity.java

```diff
@@ -752,7 +757,13 @@ public void onOrientationChanged(int orientation) {
 
 	private void setCustomActionBar() {
 		if (activityProxy.hasProperty(TiC.PROPERTY_SUPPORT_TOOLBAR)) {
-			this.setSupportActionBar(((Toolbar) ((TiToolbarProxy) activityProxy.getProperty(TiC.PROPERTY_SUPPORT_TOOLBAR)).getToolbarInstance()));
+			try {
+				this.setSupportActionBar(((Toolbar) ((TiToolbarProxy) activityProxy.getProperty(TiC.PROPERTY_SUPPORT_TOOLBAR)).getToolbarInstance()));
+			} catch (RuntimeException e) {
+				Log.e(TAG, customToolbarExceptionMessage);
```

Comment:
Since the error message will only be used once, it's best to just define it here

```Java
Log.e(TAG, "Attempting to use Toolbar as ActionBar without disabling the default ActionBar in the current theme.\n" +
           "You must set 'windowActionBar' to false in your current theme. Or use one of the following themes:\n" +
           "[Theme.Titanium, Theme.AppCompat.Translucent.NoTitleBar, Theme.AppCompat.NoTitleBar]\n" +
           "Which have ActionBar disabled by default.");
```

I have also re-worded it 👍

Reply:
Can we do a list of theme instead of []? :-)